main.go

```diff
@@ -63,9 +74,12 @@ func main() {
 	app.Flags = flags
 	app.Action = run
 	app.Version = version + " (" + gitCommit + ")"
-
+	app.Before = func(clx *cli.Context) error {
+		logger = log.New(debug)
+		return nil
+	}
 	if err := app.Run(os.Args); err != nil {
-		klog.Fatal(err)
+		logger.Fatalw("Failed to run k3k controller", zap.Error(err))
```

Comment:
```suggestion
		logger.Fatalw("failed to run k3k controller", zap.Error(err))
```
Lower casing here since it would conform to the rest of the messages.